### PR TITLE
Adds Norinori the Onigiri Costume

### PR DESCRIPTION
**Pull request by Ben10083**
Adds Norinori, the mascot of Up!Burger, costume

Sprites by Nautical 
image

### CODE REVIEW

**Comment by Ben10083** [comment]
!review

**Comment by Ben10083** [comment]
@NobleRow stated that plans are for this to be available before the event Sunday, if possible

File: code/modules/holidays/halloween/costumes.dm [hunk 1]
```diff
@@ -79,6 +79,16 @@
 	body_parts_covered = HEAD|UPPER_TORSO|LOWER_TORSO|ARMS|HANDS|LEGS|FEET
 	flags_inv = HIDEJUMPSUIT|HIDEWRISTS|HIDEFACE|HIDEEARS|BLOCKHAIR
 
+/obj/item/clothing/suit/upburger
+	name = "UP!Burger mascot costume"
```
Comment (Generalcamo) [inline code/modules/holidays/halloween/costumes.dm:83]:
```suggestion
	name = "\improper UP!Burger mascot costume"
```

File: code/modules/holidays/halloween/costumes.dm [hunk 2]
```diff
@@ -79,6 +79,16 @@
...
+	desc = "A costume of of Norinori the Onigiri, popular mascot of Up!Burger. Some poor high school student's worst part time job."
```
Comment (Generalcamo) [inline code/modules/holidays/halloween/costumes.dm:84]:
```suggestion
	desc = "A costume of Norinori the Onigiri, popular mascot of Up!Burger. Some poor high school student's worst part time job."
```

**Review by Generalcamo — CHANGES_REQUESTED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Review by Generalcamo — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**